eel/eel-canvas.c

```diff
@@ -4371,6 +4356,12 @@ eel_canvas_item_accessible_get_type (void)
         {
             return G_TYPE_INVALID;
         }
+        else if (parent_atk_type == ATK_TYPE_NO_OP_OBJECT)
+        {
+            /* use at least AtkGObjectAccessible as basis if nothing else
+             * is advertized in the registry */
+            parent_atk_type = ATK_TYPE_GOBJECT_ACCESSIBLE;
```

Comment:
This is required because ATK doesn't register an accessible type for `G_TYPE_INITIALLY_UNOWNED` in the registry, so we get an `ATK_TYPE_NO_OP_OBJECT`, yet we want to derive from something sensible, so at least `ATK_TYPE_GOBJECT_ACCESSIBLE` (like the GTK3 version).
Without this, not only some features are implemented in a fake way, but `CajaIconCanvasItemAccessible` relies on being a type derived from `ATK_TYPE_GOBJECT_ACCESSIBLE` using some of its features (like `atk_gobject_accessible_get_object()`).